tests/parsers/SquadronParserTests.java

```diff
@@ -1,5 +1,6 @@
 package parsers;
 
+import engine.GameConstants;
```

Comment:
are we hitting a point in testing where it would be easier to do things like import engine.*; and import gameComponents.*; to just pick up everything as it comes?

Reply:
Imports are automatted I think.... maybe